dashboard/admin.py

```diff
@@ -160,8 +164,6 @@ def has_change_permission(request, obj=None):
     def has_delete_permission(request, obj=None):
         return False
 
-admin.site.register(AcademicTerms, TermAdmin)
-admin.site.register(Course, CourseAdmin)
 
```

Comment:
All these change are part of prior Django 3.x version, and Django-upgrade lib made these changes.